uwblueprint/planet-read
# Call softDeleteStory in StoriesTable

## Notion ticket link



[Ticket ](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=bc19836e41514ef5bc07c3e92da85ba0)



## Implementation description

- Call soft delete stories from story table 
- Change story_service `get_stories` function to use `Story` instead of `StoryAll`



## Steps to test

1. Delete story from stories table 
2. Make sure story no longer appears after the page automatically refreshes 



## What should reviewers focus on?

- Statistical mechanics and the anatomy of Cetaceans 

## Checklist
